package.json

```diff
@@ -1,107 +1,40 @@
 {
   "name": "game-ci",
-  "version": "0.1.0",
-  "description": "Website for projects within the GameCI group",
+  "version": "0.0.0",
   "private": true,
-  "engines": {
-    "node": "12.x"
-  },
   "scripts": {
-    "dev": "cross-env NODE_OPTIONS=\"-r esm\" next",
-    "build": "node -v && yarn update-menu && cross-env NODE_OPTIONS=\"-r esm\" next build && yarn update-search",
-    "update-menu": "cross-env NODE_OPTIONS=\"-r esm -r dotenv/config\" ts-node -r tsconfig-paths/register tools/menu/cli-generate.ts",
-    "update-search": "cross-env NODE_OPTIONS=\"-r esm -r dotenv/config\" ts-node -r tsconfig-paths/register tools/search/push-search-payload.ts",
-    "analyze": "cross-env-shell NODE_OPTIONS=\"-r esm\" ANALYZE=true next build",
-    "lint": "prettier --write . --loglevel=warn && eslint . --quiet --fix",
-    "eslint": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix",
```

Comment:
We should keep the linting setup, unless there is a new setup that covers everything already.